core/node/libp2p/rcmgr_defaults.go

```diff
@@ -64,593 +55,3 @@ func logScale(val int) int {
-	if err != nil {
-		return changes, err
-	}
-	for i, op := range patch {
-		changes[i] = fmt.Sprintf("  %s", op)
-	}
-	return changes, nil
-}
-
-// https://github.com/libp2p/go-libp2p-resource-manager/blob/v0.1.5/limit_defaults.go#L49
-const expectedDefaultLimits = `{
-  "SystemBaseLimit": {
-    "Streams": 16384,
-    "StreamsInbound": 4096,
-    "StreamsOutbound": 16384,
-    "Conns": 1024,
-    "ConnsInbound": 256,
-    "ConnsOutbound": 1024,
-    "FD": 512
-  },
-  "SystemMemory": {
-    "MemoryFraction": 0.125,
-    "MinMemory": 134217728,
-    "MaxMemory": 1073741824
-  },
-  "TransientBaseLimit": {
-    "Streams": 512,
-    "StreamsInbound": 128,
-    "StreamsOutbound": 512,
-    "Conns": 128,
-    "ConnsInbound": 32,
-    "ConnsOutbound": 128,
-    "FD": 128
-  },
-  "TransientMemory": {
-    "MemoryFraction": 1,
-    "MinMemory": 67108864,
-    "MaxMemory": 67108864
-  },
-  "ServiceBaseLimit": {
-    "Streams": 8192,
-    "StreamsInbound": 2048,
-    "StreamsOutbound": 8192,
-    "Conns": 0,
-    "ConnsInbound": 0,
-    "ConnsOutbound": 0,
-    "FD": 0
-  },
-  "ServiceMemory": {
-    "MemoryFraction": 0.03125,
-    "MinMemory": 67108864,
-    "MaxMemory": 268435456
-  },
-  "ServicePeerBaseLimit": {
-    "Streams": 512,
-    "StreamsInbound": 256,
-    "StreamsOutbound": 512,
-    "Conns": 0,
-    "ConnsInbound": 0,
-    "ConnsOutbound": 0,
-    "FD": 0
-  },
-  "ServicePeerMemory": {
-    "MemoryFraction": 0.0078125,
-    "MinMemory": 16777216,
-    "MaxMemory": 67108864
-  },
-  "ProtocolBaseLimit": {
-    "Streams": 4096,
-    "StreamsInbound": 1024,
-    "StreamsOutbound": 4096,
-    "Conns": 0,
-    "ConnsInbound": 0,
-    "ConnsOutbound": 0,
-    "FD": 0
-  },
-  "ProtocolMemory": {
-    "MemoryFraction": 0.015625,
-    "MinMemory": 67108864,
-    "MaxMemory": 134217728
-  },
-  "ProtocolPeerBaseLimit": {
-    "Streams": 512,
-    "StreamsInbound": 128,
-    "StreamsOutbound": 256,
-    "Conns": 0,
-    "ConnsInbound": 0,
-    "ConnsOutbound": 0,
-    "FD": 0
-  },
-  "ProtocolPeerMemory": {
-    "MemoryFraction": 0.0078125,
-    "MinMemory": 16777216,
-    "MaxMemory": 67108864
-  },
-  "PeerBaseLimit": {
-    "Streams": 1024,
-    "StreamsInbound": 512,
-    "StreamsOutbound": 1024,
-    "Conns": 16,
-    "ConnsInbound": 8,
-    "ConnsOutbound": 16,
-    "FD": 8
-  },
-  "PeerMemory": {
-    "MemoryFraction": 0.0078125,
-    "MinMemory": 67108864,
-    "MaxMemory": 134217728
-  },
-  "ConnBaseLimit": {
-    "Streams": 0,
-    "StreamsInbound": 0,
-    "StreamsOutbound": 0,
-    "Conns": 1,
-    "ConnsInbound": 1,
-    "ConnsOutbound": 1,
-    "FD": 1
-  },
-  "ConnMemory": 1048576,
-  "StreamBaseLimit": {
-    "Streams": 1,
-    "StreamsInbound": 1,
-    "StreamsOutbound": 1,
-    "Conns": 0,
-    "ConnsInbound": 0,
-    "ConnsOutbound": 0,
-    "FD": 0
-  },
-  "StreamMemory": 16777216
-}`
-
-// https://github.com/libp2p/go-libp2p/blob/v0.18.0/limits.go#L17
-const expectedDefaultServiceLimits = `{
```

Comment:
Where's this one now?

Reply:
I move back to use the previous checking logic, I just added a list of paths that might vary and shouldn't be checked.

Reply:
Added all previous checks.